scripts/test.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+# Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+# See the file LICENSE for licensing terms.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Directory above this script
+RELAYER_PATH=$( cd "$( dirname "${BASH_SOURCE[0]}" )"; cd .. && pwd )
```

Comment:
nit: can you make this into two lines like above

Reply:
Lets also do the same in `constants.sh`

Reply:
Fixed